### DISCUSSION TITLE
Would supporting the Steam Controller be possible?

### DISCUSSION THREAD

**Discussion by FM1337** [cathery/sys-con#35] [Controller requests]
Just asking because I'm curious

**Comment by cathery** [comment]
Totally possible, but don't expect it any time soon

**Comment by qlum** [comment]
Would be very cool though but I can imagine it not being on top of the list due to the nature of the device. (steam controller = best controller)

**Comment by Nonononoki** [comment]
It even has a gyro, I'd say best third party controller for the Switch in terms of compability

**Comment by noemu** [comment]
some resources for the steam controller:
- https://github.com/kozec/sc-controller
- https://github.com/ynsta/steamcontroller
- https://lwn.net/Articles/747174/

i've started to code a little bit. (very nice documented repository)

~~One problem with this architecture is, that one USB device supports only one switch-controller. But the wireless Adapter supports up to 4 controllers.~~

I try to get it working with one controller.

**Comment by cathery** [comment]
Thank you. Any PRs are welcome
